### PR TITLE
Import ABC from collections.abc instead of collections for Python 3.9 compatibility.

### LINKED ISSUE #484: Importing ABC from collections directly was removed in Python 3.9
https://bugzilla.redhat.com/show_bug.cgi?id=1794342

Sample instance but other instances also need to be fixed. I will raise a PR for this.

https://github.com/pysb/pysb/blob/54ff9e40ecb4835eae73287f1eb983e05b9cc960/pysb/importers/json.py#L40

### PR DESCRIPTION
**Pull request by tirkarthi**
Fixes #484

### CODE REVIEW

File: pysb/bng.py [hunk 1]
```diff
@@ -33,6 +33,10 @@
 except NameError:
     basestring = str
 
+try:
+    from collections.abc import Sequence
+except ImportError:
```
Comment (alubbock) [inline pysb/bng.py]: We've [dropped Python 2.7 support](#414) recently, so there's no need for the `try`/`except` here. Simply importing from `collections.abc` should suffice. Applies to the other instances as well.

**Comment by jmuhlich** [comment]
This has been deprecated since Python 3.3 anyway, so I'm all for fixing it.

File: pysb/bng.py (hunk 1 above)
Comment (tirkarthi) [reply pysb/bng.py]: Thanks, removed them. The `setup.py` still has the classifier Python 2 and hence I added these try/except block.

Comment (alubbock) [reply pysb/bng.py]: Thanks. We're still in the process of removing Python 2 remnants from our codebase - thanks for bringing that one to my attention!

File: pysb/simulator/cupsoda.py [hunk 2]
```diff
@@ -23,6 +23,10 @@
     import pycuda.driver as cuda
 except ImportError:
     cuda = None
+try:
+    from collections.abc import Iterable
+except ImportError:
+    from collections import Iterable
```
Comment (alubbock) [inline pysb/simulator/cupsoda.py]: One more instance to remove the try/except block from. Thanks!

Comment (tirkarthi) [reply pysb/simulator/cupsoda.py]: Thanks for the catch, removed.

**Comment by alubbock** [comment]
Thanks again, looks good!

**Pull request merged**